src/components/CreatePageSplash.tsx

```diff
@@ -1,7 +1,8 @@
 import React from "react";
+import CreateBlog from "./blogContent/CreateBlog";
 
 const CreatePageSplash = () => {
-  return <div>CreatePageSplash</div>;
+  return <CreateBlog />;
```

Comment:
Do we still need this `CreatePageSplash` component? Or can we just render the CreateBlog Page?

Reply:
Let's rename this component to be `CreateBlogPage` and the `CreateBlog` component renamed to `CreateBlogInputForm` or something?

Reply:
@CallMeCCLemon 
Updated. Also create button added